src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gaps;
+  padding: $calendar-border-vert $calendar-border-hor;
+  width:
+    ($day-size * 7) +
+    ($calendar-gaps * 6) +
+    ($calendar-border-hor * 2);
+
+  &__date {
+    box-sizing: border-box;
+    width: #{$day-size};
+    height: #{$day-size};
+
+    text-align: center;
+    line-height: 100px;
+    font-family: $day-font;
+    font-size: 30px;
+    background-color: #{$day-background};
+    border: 1px solid #{$day-border-color};
+
+    cursor: pointer;
+    transition: all $day-transform-time;
```

Comment:
It is a good tone to specify the properties you want to apply transition to instead of `all`
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x